portfolio/src/main/webapp/style.css

```diff
@@ -154,12 +153,12 @@ hr {
 
 #greeting-container {
     padding: 20px 30px;
-    margin-left: auto;
-    margin-right: auto;
+    margin: 0 auto 0 auto;
 }
 
 #login {
     margin-top: 20px;
+    margin-bottom: 30px;
```

Comment:
Nit. Like you did in the margin-right and margin-left, you can combine the margin top/bottom into margin: 20px 0 30px 0

Reply:
All margins are updated!